lib/python/ray/graph.py

```diff
@@ -0,0 +1,49 @@
+# Utilities to deal with computation graphs
+
+import ray.internal.graph_pb2
+import graphviz
+
+def graph_to_dot(computation_graph, out_filename, view=False):
+  """
+  Write rendered computation graph to file and optionally display it on screen.
+
+  Args:
+    computation_graph: protocol buffer description of the computation graph
+
+    out_filename: path to the rendered computation graph that will be stored to disk
+
+    view: If true, the result the python graphviz package will try to open the
+      reslult in a viewer
+
+  Example:
+    In ray/scripts, call "python shell.py" and paste in the following code:
+
+    x = da.zeros([20, 20])
+    y = da.zeros([20, 20])
+    z = da.dot(x, y)
+
+    ray.worker.dump_computation_graph("/tmp/linalg.binaryproto")
```

Comment:
For whatever reason, on my machine, this line fails to actually create the file `/tmp/linalg.binaryproto`, and there is no error message. If I change the filename to something in the current working directory then it works.